Makefile

```diff
@@ -82,7 +82,7 @@ test:
 	go test -mod=readonly ./...
 
 bdd-test:
```

Comment:
Interesting, it takes ~7 minutes in mine--maybe closer to 30 minutes is sufficient?